css-view-transitions-1/Overview.bs

```diff
@@ -705,31 +768,22 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 	}
 	```
 
-### <dfn>::view-transition-old( <<pt-name-selector>> )</dfn> ### {#::view-transition-old}
-
-	<div class=note>
-
-		This element is a replaced element that produced the visual representation of the “old” state taken from user-agent provided snapshots.
+### View Transition Old State Image: the ''::view-transition-old()'' pseudo-element ### {#::view-transition-old}
 
-		This is only ever a [=tree/child=] of a ''::view-transition-image-pair()'',
-		never has any [=tree/children=],
-		and is omitted if there's no “old” state to represent.
+	The <dfn>::view-transition-old()</dfn> [=pseudo-element=]
+	is an empty [=named view transition pseudo-element=]
+	that represents a visual snapshot of the “old” state as a [=replaced element=];
+	it is omitted if there's no “old” state to represent.
+	Each ''::view-transition-old()'' pseudo-element is a [=tree/child=]
+	of the corresponding ''::view-transition-image-pair()'' pseudo-element.
 
+	<div class=note>
 		'':only-child'' can be used to match cases where this element is the only element in the ''::view-transition-image-pair()''.
 
 		The appearance of this element can be manipulated with `object-*` properties in the same way that other replaced elements can be.
-
 	</div>
 
-	A [=tree-abiding pseudo-element=]
-	that is also a [=named view-transition pseudo-element=],
-	and [=tree/participates=] in a [=pseudo-element tree=].
-
-	It is selected from its [=ultimate originating element=], the [=document element=].
-
-	It is a [=replaced element=], with [=natural dimensions=] equal to the content's size.
-
-	Note: The image content is captured in [=capture the image=],
+	Note: The content and [=natural dimensions=] of the image are captured in [=capture the image=],
 	then set and updated in [=setup transition pseudo-elements=] and [=update pseudo-element styles=].
```

Comment:
```suggestion
	then set in [=setup transition pseudo-elements=].
```

For the old pseudos, we capture and set it once. Only the new pseudos need an update since their image is live.